pano/pano_stitch/scripts/stitch_panorama.py

```diff
@@ -427,12 +427,24 @@ def next(self):
 
 def read_pto(pano, pto_path):
     print("\nread_pto: %s" % pto_path)
-    pano.ReadPTOFile(pto_path)
+    try:
+        # Attempt the first method
+        ifs = hsi.ifstream(pto_path)
+        pano.readData(ifs)
```

Comment:
I suppose the different API calls are required to support different versions of hugin? Again, might be nice to explain a bit what's going on.

Reply:
that's exactly that, the newer hugin versions have a different API